ui/app/mirrors/create/mirrorcards.tsx

```diff
@@ -27,12 +27,12 @@ const MirrorCards = ({
         'Query Replication allows you to specify a set of rows to be synced via a SELECT query.',
       link: 'https://docs.peerdb.io/usecases/Streaming%20Query%20Replication/overview',
     },
-    {
-      title: MirrorType.XMin,
-      description:
```

Comment:
maybe instead of removing we can explicitly advise against its use

Reply:
I like that idea too cc @iskakaushik 
Similar to how we have a line for CDC saying it's recommended